### ISSUE TITLE
Unable to connect to anything but WG Host IP

### ISSUE THREAD

**Issue by UltraBlackLinux** [wg-easy/wg-easy#885]
**Describe the bug**
I am somehow unable to connect to anything other than the IP that WG-Easy is hosted on. I can access any port on that machine, but I cannot connect to any other device on my network, not to mention the internet.

**To Reproduce**
I have no idea, here's my config:
```yaml
wg-easy:
        image: ghcr.io/wg-easy/wg-easy
        container_name: wg-easy
        restart: unless-stopped
        volumes:
            - ./config/wireguard:/etc/wireguard
        #ports:
        #    - 51820:51820/udp
        #    - 9200:51821/tcp
        environment:
            - WG_HOST=my.domain
            - WG_PORT=51820
            - WG_DEFAULT_DNS=192.168.152.10 # This device's lan IP, for Pihole
            - UI_TRAFFIC_STATS=true
            - PORT=9200
        cap_add:
            - NET_ADMIN
            - SYS_MODULE
        network_mode: host # I added this as suggested in #516, but I still cannot connect to the internet
        #sysctls:
        #    - net.ipv4.ip_forward=1
        #    - net.ipv4.conf.all.src_valid_mark=1
```

**Expected behavior**
Be able to connect to anything other than 192.168.152.10

**Desktop (please complete the following information):**
 - OS: Linux
 - Browser: ping

**Comment by pheiduck** [comment]
It needs to be like this:

```
wg-easy:
        image: ghcr.io/wg-easy/wg-easy
        container_name: wg-easy
        restart: unless-stopped
        volumes:
            - ./config/wireguard:/etc/wireguard
        ports:
            - 51820:51820/udp
            - 9200:51821/tcp
        environment:
            - WG_HOST=my.domain
            - WG_DEFAULT_DNS=192.168.152.10 # This device's lan IP, for Pihole
            - UI_TRAFFIC_STATS=true
            - PORT=9200
        cap_add:
            - NET_ADMIN
            - SYS_MODULE
        network_mode: host # I added this as suggested in #516, but I still cannot connect to the internet
        sysctls:
            - net.ipv4.ip_forward=1
            - net.ipv4.conf.all.src_valid_mark=1
```

**Comment by UltraBlackLinux** [comment]
> ports:
>             - 51820:51820/udp
>             - 51821:51821/tcp

you cannot combine `ports` and `network_mode` . I did do some more experimentation though and now I'm at the point of being able to connect to the rest of my network. I simply removed the `network_mode` setting entirely. The problem I'm running into now is that I can connect to and ping pihole, but it's somehow not used as a DNS server, or well, it is, but all attempts at dns resolution fail entirely.
